_posts/2021-05-09-april-rubygems-update.md

```diff
@@ -8,16 +8,13 @@ author_email: laurandidi21@gmail.com
 Welcome to the RubyGems monthly update! As part of our efforts at [Ruby Together](http://rubytogether.org/), we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to [RubyGems](https://github.com/rubygems/rubygems) and [RubyGems.org](https://github.com/rubygems/rubygems.org) in April.
 
 ## RubyGems News
-This Month in RubyGems, we released a new version for **RubyGems(3.2.16)** and **Bundler(2.2.16)**. 
+This Month in RubyGems, we released new versions for **RubyGems** [v3.2.16](https://github.com/rubygems/rubygems/blob/bb93b974100e9ddff7043e648d762e8a412be04e/CHANGELOG.md#3216--2021-04-08), [v3.2.17](https://github.com/rubygems/rubygems/blob/bb93b974100e9ddff7043e648d762e8a412be04e/CHANGELOG.md#3217--2021-05-05) and corresponding versions for **Bundler** [v2.2.16](https://github.com/rubygems/rubygems/blob/bb93b974100e9ddff7043e648d762e8a412be04e/bundler/CHANGELOG.md#2216-april-8-2021) and [v2.2.17](https://github.com/rubygems/rubygems/blob/bb93b974100e9ddff7043e648d762e8a412be04e/bundler/CHANGELOG.md#2217-may-5-2021). 
 
-RubyGems saw several bug fixes and updates this month, some of which include the following:
+In addition to that, we made the following improvement and fixes:
```

Comment:
I'd say "As part of those releases" or something similar, instead of "In addition to that". The latter might confuse users into thinking the specific improvements we list have not yet been released.

Reply:
yeah, sure I can make that change too.